guides/doc-Provisioning_Guide/topics/Cloud-Azure.adoc

```diff
@@ -0,0 +1,258 @@
+[[Provisioning_Virtual_Machines_in_Azure]]
+== Provisioning Cloud Instances in Microsoft Azure Resource Manager
+{ProjectName} can interact with Microsoft Azure Resource Manager (RM), including creating new virtual machines and controlling their power management states. Only image-based provisioning is supported for creating Azure RM hosts.
+
+[[Provisioning_Virtual_Machines_in_Azure-Prerequisites_for_Azure_Provisioning]]
+=== Prerequisites for Azure RM Provisioning
+
+Before you begin, ensure that the following conditions are met:
+
+* Ensure that you have the correct permissions to create an Azure Active Directory application. Fore more information, see https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#required-permissions[Check Azure AD permissions] in the _Microsoft identity platform (Azure Active Directory for developers_ documentation.
+* You must create and configure an Azure Active Directory application and service principle to obtain Application (client) ID, Directory (tenant) ID, and Client Secret. For more information, see https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal[Use the portal to create an Azure AD application and service principal that can access resources] in the _Microsoft identity platform (Azure Active Directory for developers_ documentation. 
```

Comment:
I refer to the whole section "How to: Use the portal to create an Azure AD application and service principal that can access resources", and not to a separate procedure because information about configuring tenant ID and other things is scattered across the section. So it is better for customers to read it all.

Reply:
Ohkk makes sense.